src/translate.py

```diff
@@ -945,10 +948,11 @@ def format(self, fmt: Formatter) -> str:
         return f"{self.function.format(fmt)}({args})"
 
 
-@attr.s(frozen=True, eq=True)
+@attr.s(eq=True, hash=True)
 class LocalVar(Expression):
     value: int = attr.ib()
-    type: Type = attr.ib(eq=False)
+    type: Type = attr.ib(eq=False, hash=False)
+    alias: Optional["LocalVar"] = attr.ib(eq=False, hash=False, default=None)
```

Comment:
Thanks, and great catch about it being unused.

(I previously allowed stack vars to be `Type.any()` so they could be larger than a word, so I used this to track a warning if there were any overlaps. But I added the type restriction to avoid needing this. I'll back out this change.)

Reply:
(don't forget about this!)